vscode-build/vscode.patch

```diff
@@ -867,228 +791,245 @@ index 7d8189c50be..125b6c89d0b 100644
- 					localize('menuBarVisibility', "Control the visibility of the menu bar. A setting of 'toggle' means that the menu bar is hidden and a single press of the Alt key will show it. A setting of 'compact' will move the menu into the sidebar."),
--				'included': isWindows || isLinux || isWeb
-+				'included': false
+ 				'description': localize('zenMode.centerLayout', "Controls whether turning on Zen Mode also centers the layout.")
  			},
- 			'window.enableMenuBarMnemonics': {
- 				'type': 'boolean',
--				'default': true,
-+				'default': false,
- 				'scope': ConfigurationScope.APPLICATION,
- 				'description': localize('enableMenuBarMnemonics', "Controls whether the main menus can be opened via Alt-key shortcuts. Disabling mnemonics allows to bind these Alt-key shortcuts to editor commands instead."),
--				'included': isWindows || isLinux
-+				'included': false
+ 			'zenMode.showTabs': {
+@@ -815,7 +815,7 @@ const registry = Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Con
  			},
- 			'window.customMenuBarAltFocus': {
+ 			'zenMode.hideLineNumbers': {
  				'type': 'boolean',
 -				'default': true,
 +				'default': false,
- 				'scope': ConfigurationScope.APPLICATION,
- 				'markdownDescription': localize('customMenuBarAltFocus', "Controls whether the menu bar will be focused by pressing the Alt-key. This setting has no effect on toggling the menu bar with the Alt-key."),
--				'included': isWindows || isLinux
-+				'included': false
+ 				'description': localize('zenMode.hideLineNumbers', "Controls whether turning on Zen Mode also hides the editor line numbers.")
  			},
- 			'window.openFilesInNewWindow': {
- 				'type': 'string',
+ 			'zenMode.restore': {
+diff --git a/src/vs/workbench/browser/workbench.ts b/src/vs/workbench/browser/workbench.ts
+index b0688133537..4353093fd5b 100644
+--- a/src/vs/workbench/browser/workbench.ts
++++ b/src/vs/workbench/browser/workbench.ts
+@@ -50,6 +50,8 @@ import { AccessibilityProgressSignalScheduler } from 'vs/platform/accessibilityS
+ import { setProgressAcccessibilitySignalScheduler } from 'vs/base/browser/ui/progressbar/progressAccessibilitySignal';
+ import { AccessibleViewRegistry } from 'vs/platform/accessibility/browser/accessibleViewRegistry';
+ import { NotificationAccessibleView } from 'vs/workbench/browser/parts/notifications/notificationAccessibleView';
++import { ICommandService } from '../../platform/commands/common/commands';
++import { isFromUtopiaToVSCodeMessage, messageListenersReady } from 'utopia-vscode-common';
+ 
+ export interface IWorkbenchOptions {
+ 
+@@ -193,6 +195,37 @@ export class Workbench extends Layout {
+ 				this.restore(lifecycleService);
+ 			});
+ 
++			// Chain off of the previous one to ensure the ordering of changes is maintained.
++			// FIXME Do we still need this?
```

Comment:
I don't think we still need this. IIRC the whole reason for this was to ensure writes to IndexedDB were applied in the correct order, and that required this setup because the writes would use a sequence of `await` steps (to check if the file existed, to check if the directory existed and create it if not). Now that's no longer an issue, I'm fairly certain we can remove the project changes coordination, so I'll remove it in a follow up PR